### PR TITLE
Fix issue #20 Orientation resolved

### PR DESCRIPTION
**Pull request by RockingRishav**
Fix issue #20
Todo app orientation

### CODE REVIEW

**Comment by Rohit1173** [comment]
@RockingRishav you have tried well ,appreciate your efforts but thats not how we do.
 Once run the app and check in landscape mode .You should be getting the UI of `activity_main.xml (land)` but by your pr we are not getting the same UI
A hint for you : If an app has different layouts for a single screen(i.e., potrait and landscape files) the views should contain same ids

**Comment by RockingRishav** [comment]
accha sir layout our layout(land) me jo common xml fles hain uski id same krni hai?

**Comment by Rohit1173** [comment]
> accha sir layout our layout(land) me jo common xml fles hain uski id same krni hai?

try doing it

**Comment by RockingRishav** [comment]
sir but then constraints will get deleted and then fir se constrints same karna padega?

**Comment by Rohit1173** [comment]
> sir but then constraints will get deleted and then fir se constrints same karna padega?

which constraints?? you have to change the id everywhere an that's it

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,11 +9,12 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.TODO20">
-        <activity android:name=".MainActivity">
+        <activity android:name=".MainActivity" android:configChanges="keyboardHidden|orientation|screenSize">
```
Comment (Rohit1173) [inline app/src/main/AndroidManifest.xml:12]: remove this

**Comment by Rohit1173** [comment]
LGTM,Thanks!

**Pull request merged**